src/Microsoft.Diagnostics.Monitoring.RestServer/StreamingLogger.cs

```diff
@@ -84,40 +84,47 @@ private void LogJson<TState>(LogLevel logLevel, EventId eventId, TState state, E
             //CONSIDER Should we cache up the loggers and writers?
             using (var jsonWriter = new Utf8JsonWriter(outputStream, new JsonWriterOptions { Indented = false }))
             {
+                // Matches the format of JSONConsoleFormatter
+
                 jsonWriter.WriteStartObject();
+                // TODO: Write timestamp as string
+                //jsonWriter.WriteString("Timestamp", DateTime.UtcNow.ToString());
                 jsonWriter.WriteString("LogLevel", logLevel.ToString());
-                jsonWriter.WriteString("EventId", eventId.ToString());
```

Comment:
EventId.ToString uses return Name ?? Id.ToString();
With this change, we are losing eventId.Name.

Reply:
JsonConsoleFormatter only returns the ID. Maybe we can add another property (`EventName`) for `EventId.Name` so that it remains at least a superset of JsonConsoleFormatter.

Reply:
Added EventName; it is an empty string if `EventId.Name` is not specified.